README.org

```diff
@@ -29,9 +29,10 @@ Activating ~citar-org-roam-mode~ will configure Citar to use these functions.
 
 #+begin_src emacs-lisp
 (use-package citar-org-roam
-  :after citar org-roam
-  :no-require
-  :config (citar-org-roam-mode))
+  :after (citar org-roam)
+  :config
```

Comment:
I'm thinking this, coupled with the code change below, should work if moved to the `citar` block. But I have some kind of unrelated problem that doesn't allow me to test this ATM.